scripting/practicemode.sp

```diff
@@ -22,7 +22,7 @@
 #pragma semicolon 1
 #pragma newdecls required
 
-bool g_InPracticeMode = false;
+int g_InPracticeMode = -1;
```

Comment:
I wouldn't suggest doing it this way, this leads to some fairly confusing code downstream: it's not obvious to a reader what `g_InPracticeMode=-1` means without diving into all the uses of `g_InPracticeMode`.

I would suggest instead adding a new variable, `bool g_AutostartDisabledThisMap` or such (defaulting to false), that gets set to true once the user has exited practicemode, and then checking it inside `CheckAutoStart`.

Alternatively, a 3-state enum could be defined and used the way you're doing it now, but that feels unnecessary to me.

Reply:
done, went with `g_PracticeModeCanBeAutoStarted`